roles/advanced-core/advanced_dhcp_server/readme.rst

```diff
@@ -57,21 +85,98 @@ And ice1-2:
 
 shared_network variable is optional and is simply ignored if not set.
 
-**opt 61 and opt 82**
-"""""""""""""""""""""
+opt 61 and opt 82
+"""""""""""""""""
 
-It is possible to use advanced dhcp features to identify an host. The following parameters are available, for the host and its BMC. Note that only one of these must be set for an host/BMC at the same time:
+It is possible to use advanced dhcp features to identify an host. The following
+parameters are available, for the host and its BMC. Note that only one of these
+must be set for an host/BMC at the same time:
```

Comment:
this note is no longer valid as the "multiple entries per host" functionality is provided by this PR

Reply:
You are right, I will correct it. Thanks a lot!